source/_posts/2019-04-15-hassio-ingress.markdown

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: "Introducing Hass.io Ingress"
+description: "Ingress allows Hass.io add-ons to seamlessly integrate their user interface with Home Assistant"
+date: 2019-04-15 00:00:00
+date_formatted: "April 15, 2019"
+author: Pascal Vizeli
+author_twitter: pvizeli
+comments: true
+categories: Announcements
```

Comment:
Announcements or should this be Release Notes?

Reply:
I think `Announcements` because it's not direct bounded to a single release?